stories/topics/proc-aap-aws-remove-s3-bucket.adoc

```diff
@@ -0,0 +1,27 @@
+[id="proc-aap-aws-remove-s3-bucket"]
+
+= Removing unnecessary S3 bucket files
+
+The following procedure deletes S3 bucket files left after backup.
+
+[NOTE]
+==== 
+This action cannot be reversed.
+
+This is the backup data. 
+If you delete these files, you will not be able to restore your deployment afterwards.
+====
+
+.Procedure
+. In the AWS UI, navigate to the link:https://s3.console.aws.amazon.com/s3/buckets?region=us-east-1[S3 Bucket page].
```

Comment:
```suggestion
. In the AWS UI, navigate to the link:https://s3.console.aws.amazon.com/s3/buckets?region=us-east-1[S3 Bucket page].
```
Should the region be in the link?

Reply:
I'm not sure about this. @zkayyali812 Can you help?

Reply:
After double checking it does not need to be, but user should ensure that they are in the expected region